open-sphere-base/core/src/main/java/io/opensphere/core/net/NetworkConfigurationManagerImpl.java

```diff
@@ -140,34 +141,28 @@ public void setProxy(String url, int port)
             myPrefs.putInt(PROXY_PORT_PREF_KEY, port, this);
         }
         myPrefs.putString(PROXY_HOST_PREF_KEY, url, this);
-        notifyChanged();
     }
 
     @Override
     public void setProxyConfigUrl(String url)
     {
         myPrefs.putString(PROXY_CONFIG_URL_PREF_KEY, url, this);
-        notifyChanged();
     }
 
     @Override
     public void setProxyExclusions(String hostPatterns)
     {
         myPrefs.putString(PROXY_EXCLUSION_PATTERNS_PREF_KEY, hostPatterns, this);
-        notifyChanged();
     }
 
     @Override
     public void setUseSystemProxies(boolean use)
     {
         myPrefs.putBoolean(SYSTEM_PROXIES_ENABLED_KEY, use, this);
-        notifyChanged();
     }
 
-    /**
-     * Notify when the configuration has changed.
-     */
-    private void notifyChanged()
+    @Override
+    public void notifyChanged()
     {
         myChangeSupport.notifyListeners(listener -> listener.networkConfigurationChanged());

```

Comment:
I would rather keep the notifyChanged() calls private and encapsulated in this class.  If its to prevent multiple events being sent out I would rather use a procrastinating executor, or change the interface so there is one set method that sets all config  values.

Reply:
Okay - changed the interface to have one method instead of 4 separate ones.